certified-connectors/SmartGlobalGovernance/apiProperties.json

```diff
@@ -1,24 +1,23 @@
-{
-    "swagger": "2.0",
-    "properties": {
-        "connectionParameters": {
-            "api_key": {
-                "type": "securestring",
-                "uiDefinition": {
-                    "constraints": {
-                        "clearText": false,
-                        "required": "true",
-                        "tabIndex": 2
-                    },
-                    "description": "The KEY for this API",
-                    "displayName": "API KEY",
-                    "tooltip": "Provide your API KEY"
-                }
-            }
-        },
-        "iconBrandColor": "#007ee4",
-        "capabilities": [],
-        "publisher": "Smart Global Governance",
-        "stackOwner": "Smart Global Governance"
-    }
-}
+{
+	"properties": {
+		"connectionParameters": {
```

Comment:
@Philippe-Tivolle Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
Just did the changes...

Reply:
@Phil-SGG Just checking in on the above - it looks like the formatting that Srikanth was referencing is still there in your recent changes. Are there any questions about that feedback we could help with?

Reply:
Dear All,

Sorry for my long silence. I was - and I am still - busy on other important matters.

I don't clearly understand your remark about the identation. I use Notepad++, JSON beautifier and changed all spaces by tabs. When I edit PowerPlatformConnectors/certified-connectors/SmartGlobalGovernance/apiDefinition.swagger.json and apiProperties.json (27 days old)? I don't see any issue, neither when reading nor editing...

Could you give me more information, please?

Best regards
Philippe